Symfony/CS/Fixer/Contrib/ClassyDefinitionFixer.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
+ */
+final class ClassyDefinitionFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isAnyTokenKindsFound(Token::getClassyTokenKinds());
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, Tokens $tokens)
+    {
+        for ($index = $tokens->getSize() - 1; $index > 0;--$index) {
+            if (!$tokens[$index]->isClassy()) {
+                continue;
+            }
+            $classyStart = $tokens->getNextTokenOfKind($index, array('{'));
+            for ($j = $index + 1; $j < $classyStart; ++$j) {
+                if ($tokens[$j]->isWhitespace()) {
+                    $content = $tokens[$j]->getContent();
+                    if (false === $breakAt = strpos($content, "\n")) {
```

Comment:
Internet is full of discussion is yoda bad or not, please do not discuss it here. We follow it, just like Symfony. One may not follow it, great. It's CS.

The real matter here is not to assign inside condition. Do it before, condition is only for checking conditions ;)